README.md

```diff
@@ -20,9 +20,9 @@ npm install --save react-native-nmrangeslider-ios
 Follow the [procedure to link the library in XCode](https://facebook.github.io/react-native/docs/linking-libraries-ios.html#manual-linking) 
 or run `rnpm link react-native-nmrangeslider-ios` using [rnpm](https://github.com/rnpm/rnpm).
 
-
-Then drag and drop the "DefaultTheme7" folder from `node_modules/react-native-nmrangeslider-ios/` to your
-XCode Resources folder. Select "copy files if necessary" and link them to your project. 
+Open your project in XCode if necessary and go into `Libraries > RNNMRangeSlider.xcodeproj > NMRangeSlider` and drag
+and drop the "Slider.xcassets" to the Resources folder. In the dialog, do NOT tick "copy files if necessary" because
+the files are already part of this project.
```

Comment:
There is more that needed, if images are smaller also code tweaks are necessary. So let's leave that for now.